cluster/tigera-operator.go

```diff
@@ -0,0 +1,49 @@
+package cluster
+
+import (
+	"context"
+
+	operator "github.com/tigera/operator/api/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+func tigeraClient(config *rest.Config) (*rest.RESTClient, error) {
+	crdConfig := *config
+	crdConfig.ContentConfig.GroupVersion = &schema.GroupVersion{Group: operator.GroupVersion.Group, Version: operator.GroupVersion.Version}
+	crdConfig.APIPath = "/apis"
+	crdConfig.NegotiatedSerializer = serializer.NewCodecFactory(scheme.Scheme)
+	crdConfig.UserAgent = rest.DefaultKubernetesUserAgent()
```

Comment:
I'm surprised so much stuff needed to create a kubectl client :O

Reply:
Me too, usually it's hidden when there is an auto-generated clientset but it's not provided for the operator API, only for projectcalicoV3, which isn't what we need.